src/_guides/libraries/private-files.md

```diff
@@ -112,3 +114,16 @@ that make debugging easier.
 For more information on generating and debugging
 JavaScript source for Dart apps, see the
 [dart2js documentation]({{site.webdev}}/tools/dart2js).
+
+### *.js
```

Comment:
interesting, I wonder if this is too general - is it commonly because of "out.js", would it make sense to just exclude that file only?